pkg/spoke/spokeagent.go

```diff
@@ -425,6 +428,8 @@ func (o *SpokeAgentOptions) AddFlags(fs *pflag.FlagSet) {
 		"The period to check managed cluster kube-apiserver health")
 	fs.IntVar(&o.MaxCustomClusterClaims, "max-custom-cluster-claims", o.MaxCustomClusterClaims,
 		"The max number of custom cluster claims to expose.")
+	fs.Int32Var(&o.ClientCertExpirationSeconds, "client-cert-expiration-seconds", o.ClientCertExpirationSeconds,
```

Comment:
I think we need validate this value in the `Validate` function https://github.com/open-cluster-management-io/registration/blob/main/pkg/spoke/spokeagent.go#L431, if this value is specified, we need require a min expiration 600s

Reply:
add the validation for client-cert-expiration-seconds